torch/onnx/utils.py

```diff
@@ -509,6 +509,8 @@ def _run_symbolic_function(g, n, inputs, env, operator_export_type=OperatorExpor
                 elif n.kindOf("value") == "is":
                     value = torch.stack([torch.tensor(v) for v in n["value"]]) if n["value"] else []
                     return g.op("Constant", value_t=value)
+                elif n.kindOf("value") == "i":
```

Comment:
I am not sure a non-tensor constant should be reaching this code pathway. @jamesr66a do you know?